spec/support/simplecov.rb

```diff
@@ -2,7 +2,13 @@
 require 'simplecov-json'
 require 'simplecov-lcov'
 
-formatters = [SimpleCov::Formatter::HTMLFormatter, SimpleCov::Formatter::JSONFormatter, SimpleCov::Formatter::LcovFormatter]
+formatters = [
+  SimpleCov::Formatter::HTMLFormatter,
+  SimpleCov::Formatter::JSONFormatter,
+  SimpleCov::Formatter::LcovFormatter
+]
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
```

Comment:
Required setup for undercover - https://github.com/grodowski/undercover#setting-up-required-lcov-reporting